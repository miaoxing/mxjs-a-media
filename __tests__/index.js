import Media from '../';
import {render} from '@testing-library/react';

describe('Media', () => {
  test('basic', () => {
    const result = render(<Media/>);

    expect(result.container).toMatchSnapshot();
  });

  test('body', () => {
    const result = render(<Media>
      image
      <Media.Body>body</Media.Body>
    </Media>);

    expect(result.container).toMatchSnapshot();
  });
});
