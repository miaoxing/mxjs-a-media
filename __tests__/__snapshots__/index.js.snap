// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Media basic 1`] = `
<div>
  <div
    class="flex flexDirection-row row-toTop"
  />
</div>
`;

exports[`Media body 1`] = `
<div>
  <div
    class="flex flexDirection-row row-toTop"
  >
    image
    <div
      class="flex-1 ml-12"
    >
      body
    </div>
  </div>
</div>
`;
